automate.sh

```diff
@@ -0,0 +1,60 @@
+#! /bin/sh
+
+#COLORS
+
+none='\033[0m'
+bold='\033[01m'
+disable='\033[02m'
+underline='\033[04m'
+reverse='\033[07m'
+strikethrough='\033[09m'
+invisible='\033[08m'
+
+black='\033[30m'
+red='\033[31m'
+green='\033[32m'
+orange='\033[33m'
+blue='\033[34m'
+purple='\033[35m'
+cyan='\033[36m'
+lightgrey='\033[37m'
+darkgrey='\033[90m'
+lightred='\033[91m'
+lightgreen='\033[92m'
+yellow='\033[93m'
+lightblue='\033[94m'
+pink='\033[95m'
+lightcyan='\033[96m'
+
+
+# Variables for folder names and paths 
+BUILD_F=build
+OUTPUT_F=build/output
+
+PATH_ONE=/root/libtorch
+PATH_TWO=/root//libtorch/share/cmake/Torch/
+
+
+
+if [ -d $BUILD_F ] ; then
+	echo "${bold}${yellow}build folder Already Present... !! skipping folder creation"
+else
+	echo "${bold}${purple}Creating ${bold}build folder"
+	mkdir build && echo "${bold}${lightgreen}Done" || (echo "${bold}${red}Folder creation failed.. exiting..." && exit 0)
+fi
+
+
+echo "${none}Running ${bold}${lightblue}Cmake ${none}inside the ${bold}${green}build ${none}folder..."
+(cd build && cmake -DCMAKE_PREFIX_PATH=${PATH_ONE} -DCMAKE_MODULE_PATH=${PATH_TWO} ..) || (echo "${bold}${red}cmake command failed... exiting..${none}" && exit 0)
```

Comment:
Do we need to pass OpenCV_DIR path as well (in cmake command)? Something like:

```bash
cmake -DCMAKE_PREFIX_PATH=${PATH_ONE} -DCMAKE_MODULE_PATH=${PATH_TWO} -DOpenCV_DIR=/root/opencv/build/ ..
```

Reply:
i don't know if not passing the path to opencv was affecting the execution in any way.... but it's always better to add the path .....

Reply:
Building OpenCV links libraries to our system, so using PATH for OpenCV isn't necessary. But it's always better to add the path, as you said :)